Helper/Cart.php

```diff
@@ -1592,8 +1591,11 @@ public function buildCartFromQuote($quote, $immutableQuote, $items, $placeOrderP
         $billingAddress  = $immutableQuote->getBillingAddress();
         $shippingAddress = $immutableQuote->getShippingAddress();
 
-        //Use display_id to hold and transmit, all the way back and forth, both reserved order id and immutable quote id
-        $cart['display_id'] = $immutableQuote->getReservedOrderId() . ' / ' . $immutableQuote->getId();
+        //Use display_id to hold and transmit, all the way back and forth, reserved order id
+        $cart['display_id'] = $immutableQuote->getReservedOrderId();
```

Comment:
As we discussed in slack, we need to stop reserve order id and leave display_id empty at this moment.

Reply:
Let's not rush with this and go one step at a time. The changes presented in this PR are more like a refactor and should not affect the logic and data flow.

What you are suggesting would affect the legacy order creation and probably have side effects on hooks logic (for payment declined or failed, for example). That's a big change, although may not look like it in terms of code diff.

It should be done in a separate PR. Whoever does it, should thoroughly run ALL of the hook type / various credit card test cases from the production readiness tests spreadsheet with BOTH pre-auth and legacy flow before the pull request is made.  Thinking of every possible scenario is developers' job (who know how things actually work), QA is there just to verify. 

I think that's the most professional way of doing it. And probably the fastest in the long run. Pushing ad hoc, untested changes (from all aspects), and waiting for QA to find regressions (hopefully before merchants do) doesn't seem to speed us up in the recent practice.